playbooks/roles/configure_lbaasv2_agent/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+# version: 2016-12-20
+---
```

Comment:
Can you add a comment to this file simply to state that the agent ini contains many settings, but this file is starting with the most commonly used settings?  (and feel free to add more as required)